src/domain/core/src/entities/metadata_chain.rs

```diff
@@ -22,7 +24,7 @@ use crate::repos::{SetRefError as SetRefErrorRepo, *};
 #[async_trait]
 pub trait MetadataChain: Send + Sync {
     /// Resolves reference to the block hash it's pointing to
-    async fn get_ref(&self, r: &BlockRef) -> Result<Multihash, GetRefError>;
+    async fn get_hash_by_ref(&self, r: &BlockRef) -> Result<Multihash, GetRefError>;
```

Comment:
PS: I would suggest `resolve_ref` - short and consistent with `resolve_dataset_ref`

Reply:
> Note for future: it's better separate mass-renamings like this one into a separate PR or agree on them verbally and make them just before merging the branch. 
> This would save a lot of time for reviewers, as I'm struggling to understand how caching is actually implemented in the sea of ~50 files that were changed.

Yes, you're right, of course -- next time such changes will be put in separate PRs. PRs must be atomic

Reply:
> PS: I would suggest resolve_ref - short and consistent with resolve_dataset_ref

will use

Reply:
Done via https://github.com/kamu-data/kamu-cli/pull/505